apps/tinyheads/lib.js

```diff
@@ -135,7 +135,13 @@ exports.drawFace = function(scale, eyesNum, mouthNum, peek, offset) {
   // Draw face
   let xOffset = (g.getWidth() - (exports.faceW * scale)) / 2;
   let yOffset = (offset ? offset : 0) + ((g.getHeight() - (exports.faceH * scale)) / 2);
-  g.setBgColor(0, 0, 0);
+  
+  if (exports.settings.showWidgets == 'on' || (exports.settings.showWidgets == 'unlock' && !Bangle.isLocked())) {
```

Comment:
This will also cause the background to change to the hair colour when editing the tinyhead, but we can work around  this by checking the scale value so we still use a black background for the smaller editor scale. Bit of a hack but it'll do.

`if (scale == 9 && (exports.settings.showWidgets == 'on' || (exports.settings.showWidgets == 'unlock' && !Bangle.isLocked()))) {`

Reply:
good point, it should be fixed now